lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -38,6 +38,8 @@ def run_loop
           rescue EOFError => e
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
+          ensure
+            terminate
```

Comment:
I guess so. Since this is supposed to be a _forever_ loop may be we will never have successful exit.

I have tested the fix in my local setup. Looks like it is working.

BTW: Will there be a new release of the gem?